lib/liquid.rb

```diff
@@ -24,6 +24,8 @@ module Liquid
   ArgumentSeparator           = ','
   FilterArgumentSeparator     = ':'
   VariableAttributeSeparator  = '.'
+  WhiteSpaceControl           = /-/
+  WhiteSpace                  = /[ \t]*\r?\n?[ \t]*/
```

Comment:
I'm not saying a full trim is more useful.  It is just simpler.  If it isn't useful to have it stop stripping whitespace on the second newline preceding or following a tag, then why don't we just use `lstrip` and `rstrip` which are easier to document and understand.